src/main/resources/jenkins/security/plugins/ldap/LDAPConfiguration/help-ignoreIfUnavailable.html

```diff
@@ -0,0 +1,11 @@
+<div>
+    <p>
+    If checked, this option indicates that authentication and user/group membership lookups
+    will be attempted against subsequent LDAP domains even if this domain is unavailable.
+    <p>
+    If unchecked (the default), authentication and user/group memberships for users in
+    subsequent LDAP domains will fail if this domain is unavailable.
```

Comment:
> will fail

"will not even be attemped" no ?

Reply:
Yes I will reword this, I should also change `memberships` to `membership lookups`. Thanks!